plans/distribution-keylime-tests-github-ci.fmf

```diff
@@ -0,0 +1,18 @@
+summary:
+  Tests used by Packit/TFT CI on Github to test distribution keylime
+adjust:
+    enabled: false
+    when: distro = centos-stream
```

Comment:
Do we want to enable it also for CentOS Stream? We have bulds for them as well in the copr repo.

Reply:
> Do we want to enable it also for CentOS Stream? We have bulds for them as well in the copr repo.

Are those in par with Fedora packages?

Reply:
Always is a hard statement :) But yeah, the idea is that they will be -- I have to build there also when I build new updates in Fedora